### DISCUSSION TITLE
Get Coin Analytics

### DISCUSSION THREAD

**Discussion by SKOHscripts** [horizontalsystems/unstoppable-wallet-android#6041] [Q&A]
Hi everyone, 
I got the 0.31 update and I can't find where to show coin analytics as implemented in a previous update. I was able to show MACD, RIS .. but I can't any more. 
Any idea ? Where can I find it in the app ?

**Comment by serrq** [comment]
Same observation. Where are the enhanced coins analytic?

**Reply by SKOHscripts** [reply]
Came back to 0.30 and everything is fine now. Seems to have a pb with 0.31 and coins analytics.

**Comment by Dianaaiym** [comment]
> Hi everyone, I got the 0.31 update and I can't find where to show coin analytics as implemented in a previous update. I was able to show MACD, RIS .. but I can't any more. Any idea ? Where can I find it in the app ?

We would like to return chart indicators, and we need feedback from users. Please email us (support.unstoppable@protonmail.com) if you have any suggestions.

**Comment by Wjxfi** [comment]
I need that too!
